requirements.txt

```diff
@@ -19,8 +19,9 @@ tldextract==2.2.*
 urllib3==1.25.*
 matplotlib==3.1.*
 pandas==0.25.*
-seaborn==0.10.*
+seaborn>=0.10.*
```

Comment:
Perhaps set the requirement to `seaborn==0.11.*` so that we can get bug fixes but not minor / major version upgrades as Seaborn seems to have breaking changes even between minor version upgrades.